src/pages/EditMonitor.vue

```diff
@@ -64,6 +64,9 @@
                                         <option value="mqtt">
                                             MQTT
                                         </option>
+                                        <option value="rabbitmq">
+                                            {{ $t("RabbitMQ (using Management Plugin)") }}
```

Comment:
Sorry, but I still don't think this is as clear as it can be.

In my mind, a helptext (including a link to the rabitmq-docs) below the input where you can add the node urls would be much more clear.
What do you think?

[*(docs for the feature in our i18n library)*](https://vue-i18n.intlify.dev/guide/advanced/component.html)